editor/grida-canvas-react/renderer.tsx

```
@@ -146,42 +152,23 @@ export function Transformer({ children }: React.PropsWithChildren<{}>) {
+  const documentKey = useEditorState(editor, (state) => state.document_key);
+  const sceneId = useEditorState(editor, (state) => state.scene_id);
+  const { fit } = useTransform();
+
+  useEffect(() => {
+    fit("*");
+  }, [documentKey, sceneId]);
+}
+
 export function AutoInitialFitTransformer({
   children,
 }: React.PropsWithChildren<{}>) {
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)